spec/features/users/dashboards/show_spec.rb

```diff
@@ -1,5 +1,58 @@
 require 'rails_helper'
 
 RSpec.describe 'Dashboard page' do
-  
-end
\ No newline at end of file
+  before(:each) do
+    user_1 = User.create!(email: "user1@email.com", password_digest: "1234")
+    visit '/'
+    click_on("New to Viewing Party? Register Here")
+    within 'form' do
+      fill_in 'user[email]', with: 'test2@test.com'
+      fill_in 'user[password]', with: 'password'
+      click_button 'Register'
```

Comment:
Utilize FactoryBot to create your 2 users and use `login_as(user)` helper for logging in